src/main/webapp/index.jsp

```diff
@@ -131,5 +138,12 @@
             </form>
         </div>
     </div>
+    <div class="modalWrapper" id="taskInfoModalWrapper">
+        <div class="modal" id="taskInfoModal">
+            <span class="close-button" id="task-info-close-button" onclick="closeTaskInfoModal()">&times;</span>
```

Comment:
Same comment as above about the onclick function in here.

Reply:
Sure. Can you help me change this onclick function? Thanks.

Reply:
Yup! I'll make these changes, just opened a new issue for it #86